api/HTMLLinkElement.json

```diff
@@ -143,6 +190,342 @@
           }
         }
       },
+      "disabled": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "1",
+              "notes": "In Chrome and other Blink-based browsers, adding the <code>disabled</code> attribute using JavaScript does not remove the stylesheet from <code>document.styleSheets</code>."
+            },
+            "chrome_android": {
+              "version_added": "18",
+              "notes": "In Chrome and other Blink-based browsers, adding the <code>disabled</code> attribute using JavaScript does not remove the stylesheet from <code>document.styleSheets</code>."
+            },
+            "edge": {
+              "version_added": "12",
+              "notes": "Since Edge 79, adding the <code>disabled</code> attribute using JavaScript does not remove the stylesheet from <code>document.styleSheets</code>."
```

Comment:
Did you check if this works in Edge 18? I think the correct per-spec behavior for `disabled` was only implemented by Firefox, and suspect Edge 18 might have had the same problem as Chromium here.

Reply:
Yep, looks like this works correctly in Edge 18, so the behavior started in Edge 79.

Test Code:
```html
<!DOCTYPE html>
<html>
	<head>
		<meta content="width=device-width">
		<style id="persistent-style">
			/* Persistent styling */
			body {
				background-color: black;
				color: white;
				margin: 0;
				padding: 0;
			}
			canvas {
				background-color: white;
			}
		</style>
		<style id="test-style">
			@media (max-width: 600px) {
				body {
					background-color: #111;
					color: white;
				}
			}
		</style>
		<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" disabled />
	</head>
	<body>
		<div id="test"></div>

		<script>
			alert(document.styleSheets.length + " / 3");
		</script>
	</body>
</html>
```